js/ui/text_box/texteditor_button_collection/custom.js

```diff
@@ -30,6 +30,7 @@ export default class CustomButton extends TextEditorButton {
         this._addToContainer($element);
 
         const instance = editor._createComponent($element, Button, extend({}, this.options, {
+            parentIndependentState: true,
```

Comment:
Let's pick up another name for the option

Reply:
Renamed to ignoreParentReadOnly